pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -79,6 +79,8 @@ message DNSOption {
     repeated string servers = 1;
     // List of DNS search domains of the cluster.
     repeated string searches = 2;
+    // List of DNS opt strings
+    repeated string opts = 3;
```

Comment:
We change the numbers corresponding to the  fields in every PR. I think it's expected that this will be constantly broken :\